plexus-compilers/plexus-compiler-eclipse/src/test/java/org/codehaus/plexus/compiler/eclipse/EclipseJavaCompilerTest.java

```diff
@@ -14,42 +15,42 @@
 {
     @ParameterizedTest
     @MethodSource( "sources" )
-    void testReorderedSources( List<String> expected, List<String> inputSources )
+    void testReorderedSources( LinkedHashSet<String> expected, LinkedHashSet<String> inputSources )
```

Comment:
definitely a nit comment (sorry for that): maybe using Set rather than the implementation?